boards/scdz/ch32v307evt/board.cmake

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Michael Hope
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(minichlink)
```

Comment:
Just curious, did you manage to use `west flash` with minichlink ? I had to specify to minichlink to flash at `0x08000000`, using `--dt-flash=y` and putting the flash at this address instead of 0 in the devicetree.